src/applications/vaos/newAppointmentFlow.js

```diff
@@ -41,19 +51,33 @@ export default {
   },
   typeOfCare: {
     url: '/new-appointment',
-    async next(state) {
+    async next(state, dispatch) {
       let nextState = 'vaFacility';
 
       if (isSleepCare(state)) {
         nextState = 'typeOfSleepCare';
       } else if (isCommunityCare(state)) {
         try {
-          const data = await getCommunityCare(
-            '/vaos/community-care/eligibility',
+          // Check if user registered systems support community care...
+          const userSystemIds = await getSystemIdentifiers();
+          const communityCareSites = await getSitesSupportingVAR();
+          const communityCareSite = communityCareSites.find(site =>
```

Comment:
You could do `some` instead of `find` here and that would return a boolean, which might be clearer to read.